system/Commands/Generators/MigrationGenerator.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+/**
+ * This file is part of the CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Commands\Generators;
+
+use CodeIgniter\CLI\BaseCommand;
+use CodeIgniter\CLI\CLI;
+use CodeIgniter\CLI\GeneratorTrait;
+
+/**
+ * Generates a skeleton migration file.
+ */
+class MigrationGenerator extends BaseCommand
+{
+	use GeneratorTrait;
+
+	/**
+	 * The Command's Group
+	 *
+	 * @var string
+	 */
+	protected $group = 'Generators';
+
+	/**
+	 * The Command's Name
+	 *
+	 * @var string
+	 */
+	protected $name = 'make:migration';
+
+	/**
+	 * The Command's Description
+	 *
+	 * @var string
+	 */
+	protected $description = 'Generates a new migration file.';
+
+	/**
+	 * The Command's Usage
+	 *
+	 * @var string
+	 */
+	protected $usage = 'make:migration <name> [options]';
+
+	/**
+	 * The Command's Arguments
+	 *
+	 * @var array
+	 */
+	protected $arguments = [
+		'name' => 'The migration class name.',
+	];
+
+	/**
+	 * The Command's Options
+	 *
+	 * @var array
+	 */
+	protected $options = [
+		'--session'   => 'Generates the migration file for database sessions.',
+		'--table'     => 'Supply a table name.',
+		'--dbgroup'   => 'Database group to use, Default: "default".',
```

Comment:
I think using comma to separate two strings on the same line is more correct than using dot

Reply:
This one is based on grammar. When you combine two phrases with a comma, there must be perceived continuity of thought. But since these are not full sentences, you cannot use comma to combine them so you use a period. Commas will be "correct" if you structured the options descriptions like this:

Database group to use for database sessions, and defaults to "default".

Reply:
And when you read it vs how it looks, the comma placement will make it not "sound right", which is a mysterious rule English has.

Reply:
fair enough